relation-extraction/src/main/java/edu/illinois/cs/cogcomp/pipeline/handlers/StanfordTrueCaseHandler.java

```diff
@@ -0,0 +1,72 @@
+/**
```

Comment:
This is duplicate; right?

Reply:
I mean yeah, the same one is also included in MD. But this class is a part of another package (neither MD or RE), I am reloading this class here, so I think I have to include it under src/ ? Let me know if there is another way.

Reply:
This is using an older version of stanford; right? If so, why not move it to [here](https://github.com/CogComp/cogcomp-nlp/tree/master/external/stanford_3.3.1/src/main/java/edu/illinois/cs/cogcomp/pipeline/handlers)?

Reply:
Sounds good